src/features/Header/index.tsx

```diff
@@ -49,11 +65,10 @@ export default function Header() {
             pr={useExtraNarrowGap ? extraNarrowGap : `${slotsNavSpacing}px`}
             pb={`${headerBottomSpacing}`}
```

Comment:
nit: I know you didn't change this but I just noticed

```suggestion
            pb={headerBottomSpacing}
```